lib/elixir/lib/string.ex

```diff
@@ -1441,50 +1441,95 @@ defmodule String do
   end
 
   @doc """
-  Returns an integer representing Levenshtein distance between
-  `source` and `target`.
+  Returns a float value between 0 (equates to no similarity) and 1 (is an exact match)
+  representing [Jaro](http://en.wikipedia.org/wiki/Jaro–Winkler_distance)
+  distance between `str1` and `str2`.
 
-  Levenshtein distance between two words is the minimum number of
-  single-character edits (i.e. insertions, deletions or substitutions)
-  required to change one word into the other.
-
-  The algorithm used is based on the [Wagner-Fisher iterative
-  implementation](http://en.wikipedia.org/wiki/Levenshtein_distance#Iterative_with_two_matrix_rows).
+  The Jaro distance metric is designed and best suited for short strings such as person names.
 
   ## Examples
 
-      iex> String.levenshtein_distance("kitten", "sitting")
-      3
+      iex> String.jaro_distance("dwayne", "duane")
+      0.8222222222222223
+      iex> String.jaro_distance("even", "odd")
+      0.0
+
   """
-  @spec levenshtein_distance(t, t) :: non_neg_integer
-  def levenshtein_distance(source, target)
 
-  def levenshtein_distance(source, source), do: 0
+  @spec jaro_distance(t, t) :: 0..1
```

Comment:
@eksperimental right, we should fix this but for now it could only be `float` (37e4f20bd67f3190e7aff07f687d041065bdd1e6).
And seems [elixir-lang/gettext](https://github.com/elixir-lang/gettext/blob/master/lib/gettext/fuzzy.ex) need this correction as well (could you open PR there? :)).